board/piksiv3/rootfs-overlay/etc/init.d/S86skylark0_setup

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+name="skylark_setup"
+
+start() {
+  # Setup DNS
+  # TODO (mookerji): Move to a template command?
+  rm -f /etc/resolv.conf
+  echo "nameserver 8.8.8.8" > /etc/resolv.conf
+  date -s '2017-03-09 16:14:00'
+
+  # Roll your own formatted UUID for Skylark client
```

Comment:
I avoided doing this originally because the UUID-logic is used in one place and isn't really accessible/queryable through the onboard Linux services: https://github.com/swift-nav/piksi_firmware_private/blob/46acdc0fe1cecd9544e569d79087f41681e8ab25/src/board/v3/init.c#L301. I was mimicking the pattern of assuming this data was accessible in the filesystem somewhere.

is your preference that I (a) have something similar in libskylark to what's in the firmware already, or ...?